Cookbook/Current.md

```diff
@@ -0,0 +1,18 @@
+# How to use `Current` in Babylon iOS Project
+
+### Where is that? 
+- `Current` is the global instance of the `World` type which is located in the `BabylonDependencies.framework`. 
+It's purpose is to provide convenient access to the instances that should be shared between different parts of the app (cross-cutting concerns). For more information please read the [How Control the World](/Cookbook/Proposals/ControlTheWorld.md) document.
+
+## `Do`s and `Don't`s
+
+#### do
+- access `Current` **only** from `Builder`s layer;
+- pass the required dependencies down the chain;
```

Comment:
What does this mean? Using current to access some stuff that is then injected to a VM?

Reply:
pass the dependencies to VM and any other layer that needs any dependency. 
Isn't that clear?